app/services/processors/project_privacy_backfiller.rb

```diff
@@ -0,0 +1,9 @@
+module Processors
+  class ProjectPrivacyBackfiller < BaseService
+    def call
+      Project.find_each do |project|
+        project.update(is_private: project.events.first&.data&.dig('repository', 'private'))
```

Comment:
Will `:dig` work even if `first&` or `data&` are nil? `first&` will catch a nil value but I am sure about `nil.dig`

Reply:
I think that's the point in using `&`
If `project.events.first&.data` is `nil`, then `dig('repository', 'private')` will return `nil` as well

Reply:
I answer myself, never mind. It won't get called if `:first` is nil.